scripts/qa_crowbarsetup.sh

```diff
@@ -449,6 +449,21 @@ function add_ha_repo()
     done
 }
 
+function add_ha12_repo()
+{
+    # TODO: add Updates-Test repo
+    local repo
+    for repo in SLE12-HA-{Pool,Updates}; do
+        if [ "$hacloud" == "2" -a "$repo" == "SLE12-HA-Updates-test" ] ; then
```

Comment:
Right, I "kept" them (kept, because it's copied from another function) because of the `TODO`. I can drop that if you feel strong about it, but I wanted to make the code ready for the Updates-Test case.